ui/src/components/Projects/Create/index.tsx

```diff
@@ -0,0 +1,204 @@
+import classNames from 'classnames/bind';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import Link from 'next/link';
+import { useEffect } from 'react';
+import { AiOutlineTeam } from 'react-icons/ai';
+import { MdAdd } from 'react-icons/md';
+import * as Yup from 'yup';
+
+import ImageUploadField from 'src/components/Form/ImageUploadField';
+import SelectField from 'src/components/Form/SelectField';
+import TextField from 'src/components/Form/TextField';
+import NoItemsFound from 'src/components/NoItemsFound';
+import TeamsList from 'src/components/Projects/TemsList';
+import { Organization } from 'src/entities/Organizations';
+
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Props {
+  organizations: Organization[];
+  isFetchingOrgs: boolean;
+  values?: CreateProjectFormValues;
+  isEditMode?: boolean;
+  isFetchingProject?: boolean;
+  projectId?: string;
```

Comment:
`projectId` here can help us distinguish b/w edit vs create mode. `isEditMode` or equivalent will be helpful when we add "Edit" action for this view.